test/foundry/CacheManager.t.sol

```diff
@@ -134,11 +161,7 @@ contract ArbOwnerPublicMock {
 contract ArbWasmMock {
     // returns a non-uniform distribution of mock code sizes
     function codehashAsmSize(bytes32 codehash) external pure returns (uint64) {
-        uint256 size;
-        for (uint256 i = 0; i < 3; i++) {
```

Comment:
This created a triangle distribution of sizes, which is more representative of prod than a uniform distribution. Probs doesn't matter though

Reply:
I see, won't fix.